homeassistant/components/samsungtv/strings.json

```diff
@@ -44,5 +44,15 @@
     "trigger_type": {
       "samsungtv.turn_on": "Device is requested to turn on"
     }
+  },
+  "issues": {
+    "deprecated_wol": {
+      "title": "The SamsungTV wake-on-lan functionnality is being removed",
+      "description": "Automatic wake-on-lan for SamsungTV is being removed.\n\nReconfigure to use the turn_on trigger."
```

Comment:
Reconfigure what? How? :)

The repair issues are aimed towards the end-user, we should carefully write those as such.

Reply:
I have updated with the following:

![image](https://user-images.githubusercontent.com/6771947/225726686-155ba94d-edef-43f3-89f0-84e594729118.png)